src/libos/src/events/waiter.rs

```diff
@@ -154,7 +162,7 @@ impl Inner {
     pub fn wake(&self) {
         if self
             .is_woken
-            .compare_exchange(false, true, Ordering::SeqCst, Ordering::SeqCst)
+            .compare_exchange(false, true, Ordering::Acquire, Ordering::Relaxed)
```

Comment:
Should it be `AcqRel` as the compare_exchange is a load-store-combined operation?